src/main/webapp/resources/js/pages/projects/ncbi/create/CreateNcbiExport.tsx

```diff
@@ -0,0 +1,443 @@
+                            name="organization"
+                            rules={[
+                              {
+                                required: true,
+                                message: i18n(
+                                  "NcbiSubmissionRequest.organization.description"
+                                ),
+                              },
+                            ]}
+                            label={i18n("NcbiSubmissionRequest.organization")}
+                            help={i18n(
+                              "NcbiSubmissionRequest.organization.description"
+                            )}
+                          >
+                            <Input />
+                          </Form.Item>
+                        </Col>
+                        <Col md={12} xs={24}>
+                          <Form.Item
+                            name="namespace"
+                            rules={[
+                              {
+                                required: true,
+                                message: i18n(
+                                  "NcbiSubmissionRequest.namespace.description"
+                                ),
+                              },
+                            ]}
+                            label={i18n("NcbiSubmissionRequest.namespace")}
+                            help={i18n(
+                              "NcbiSubmissionRequest.namespace.description"
+                            )}
+                          >
+                            <Input />
+                          </Form.Item>
+                        </Col>
+                        <Col md={12} xs={24}>
+                          <Form.Item
+                            name="releaseDate"
+                            rules={[
+                              {
+                                required: true,
+                                message: i18n(
+                                  "NcbiSubmissionRequest.releaseDate.description"
+                                ),
+                              },
+                            ]}
+                            label={i18n("NcbiSubmissionRequest.releaseDate")}
+                            help={i18n(
+                              "NcbiSubmissionRequest.releaseDate.description"
+                            )}
+                          >
+                            <DatePicker
+                              style={{ width: "100%" }}
+                              disabledDate={disabledDate}
+                            />
+                          </Form.Item>
+                        </Col>
+                      </Row>
+                    </Card>
+                    <Card title={i18n("CreateNcbiExport.samples")}>
+                      {invalid.length > 0 && (
+                        <Alert
+                          type="warning"
+                          showIcon
+                          message={"Some Samples are not valid"}
```

Comment:
Internationalize this text

Reply:
Updated in 8397566de0a6b2592b96ab58382ddcba08279861, also displays a list of the invalid sample names